manifests/03_delete_stale_objects.yaml

```diff
@@ -0,0 +1,147 @@
+# These 4 objects were added in 4.1 and removed in 4.6.
+# The release.openshift.io/delete annotation will remove them
+# from clusters that had them installed by previous releases.
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-storage-operator
+  namespace: openshift-cluster-storage-operator
+  annotations:
+    release.openshift.io/delete: "true"
```

Comment:
You probably also need the cluster-profile annotations like:

```yaml
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
```

otherwise the CVO will completely ignore the manifest.

Reply:
Thank you! I was scratching my head why the manifest wasn't getting applied during testing.

Reply:
@wking After adding those cluster-profile annotations, the manifest is getting applied. However... the objects are not getting deleted. I'm wondering what I'm missing.
I tested 2 scenarios:
1) fresh install of 4.7.18 (does not include the original manifests), then upgrade to a release with these changes.
2) apply the original manifests to a new 4.7.18 cluster, then upgrade to a release with these changes.
In case 1, the objects got created with the delete annotation. In case 2, the delete annotation was added to the existing objects. But in both cases, those objects were never deleted. Is this expected? What actually triggers CVO to delete objects based on the annotation?